lib/session/sessionManager.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { OnSessionMessage, SessionHandlerOptions, MessageSession } from "./messageSession";
+import { OnError } from "../core/messageReceiver";
+import { ClientEntityContext } from "../clientEntityContext";
+import { getProcessorCount } from "../util/utils";
+import * as log from "../log";
+
+export class SessionManager {
+  /**
+   * @property {boolean} isManagingSessions Indicates whether the manage is currently managing
+   * sessions.
+   * - **Default**: `false`.
+   */
+  isManagingSessions: boolean = false;
+  /**
+   * @property {number} maxConcurrentSessions The maximum number of sessions that the user wants to
+   * handle concurrently.
+   * - **Default**: `2000`.
+   */
+  set maxConcurrentSessions(value: number) {
```

Comment:
`maxConcurrentSessions` only seems to be used to set `_maxConcurrentAcceptSessionRequests`.
In which case, why even track `maxConcurrentSessions` as a public property?

Reply:
Not really. I still need to add code to SessionManager. Which will make this a bit clearer. As mentioned above SessionManager code as seen until now is just a placeholder.